### PR TITLE
Mobile Release v1.76.1

### PR DESCRIPTION
**Pull request by geriux**
## Description
Release 1.76.1 of the react-native-editor and Gutenberg-Mobile.

For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/4875

### CODE REVIEW

File: packages/editor/src/components/editor-help/move-blocks.native.js [hunk 1]
```diff
@@ -12,19 +12,39 @@ import { __ } from '@wordpress/i18n';
...
 	return (
 		<>
+			<HelpDetailImage
+				source={ require( './images/drag-and-drop-light.png' ) }
+				sourceDarkMode={ require( './images/drag-and-drop-dark.png' ) }
+			/>
+			<View style={ styles.helpDetailContainer }>
+				<HelpDetailSectionHeadingText
+					text={ __( 'Drag & drop' ) }
+					badge="NEW"
```
Comment (fluiddot) [inline packages/editor/src/components/editor-help/move-blocks.native.js]: Yikes, I'm afraid I totally overlooked translating this string 🤦 . I'll open a PR for addressing it as soon as possible.

Comment (geriux) [reply packages/editor/src/components/editor-help/move-blocks.native.js]: 😅 I totally missed it too!

Comment (fluiddot) [reply packages/editor/src/components/editor-help/move-blocks.native.js]: I fixed this in https://github.com/WordPress/gutenberg/pull/41196/commits/d0b2cea01f9d12ae384ea8f1c9d32c0664cad5b8.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

File: packages/react-native-editor/ios/Podfile.lock [hunk 2]
```diff
@@ -542,10 +542,10 @@ SPEC CHECKSUMS:
...
   RNSVG: 36a7359c428dcb7c6bce1cc546fbfebe069809b0
-  RNTAztecView: 0783a43eb5241e38587a52cb4eadc0ef2852607a
+  RNTAztecView: def74944705c4bef636fc653d11eaa1f49b64c22
   WordPress-Aztec-iOS: 7d11d598f14c82c727c08b56bd35fbeb7dafb504
   Yoga: 9a08effa851c1d8cc1647691895540bc168ea65f
 
 PODFILE CHECKSUM: d0a2d4714ee19a1821eb2a30029333be8d6a729f
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.11.3
```
Comment (fluiddot) [inline packages/react-native-editor/ios/Podfile.lock]: Looks like the Cocoapods version has changed but the `Gemfile.lock` points to the previous version:
https://github.com/WordPress/gutenberg/blob/78cea0f53cc25266e9dba08e49a3af8d93cfe8bc/packages/react-native-editor/ios/Gemfile.lock#L17

I'm wondering whether we should update it 🤔 . In any case, if we do it, we should also update the `Gemfile` file too:
https://github.com/WordPress/gutenberg/blob/78cea0f53cc25266e9dba08e49a3af8d93cfe8bc/packages/react-native-editor/ios/Gemfile#L3

Comment (geriux) [reply packages/react-native-editor/ios/Podfile.lock]: Nice catch, I reverted it

**Pull request merged**